tests/unit_tests/data_steward/cdr_cleaner/cleaning_rules/deid/remove_fitbit_data_if_max_age_exceeded_test.py

```diff
@@ -33,7 +33,7 @@ def setUp(self):
         self.project_id = 'foo_project'
         self.dataset_id = 'foo_dataset'
         self.sandbox_id = 'foo_sandbox_dataset'
-        self.combined_dataset_id = 'combined_dataset'
```

Comment:
Let's keep `self.combined_dataset_id = 'combined_dataset'` as it is.

Line 39-41 should be changed like the following. Otherwise, the class's fourth argument is `table_namer` and `self.combined_dataset_id` will be assigned to `table_namer`, which causes this test to fail.
```
        self.rule_instance = RemoveFitbitDataIfMaxAgeExceeded(
            self.project_id, self.dataset_id, self.sandbox_id,
            combined_dataset_id=self.combined_dataset_id)
```